SU2_CFD/src/solvers/CIncNSSolver.cpp

```diff
@@ -809,10 +798,12 @@ void CIncNSSolver::SetTauWall_WF(CGeometry *geometry, CSolver **solver_container
           U_Tau = U_Tau - relax*(diff / grad_diff);
 
           counter++;
-
           if (counter > max_iter) {
             notConvergedCounter++;
-            cout << "Warning: Y+ did not converge within the max number of iterations!" << endl;
```

Comment:
I was trying this on the CRM and these messages were showing up a lot, so I made some changes to print only one message with totals.
I also copied the "safe values" you had for the compressible version.

Reply:
Hi Pedro, Yes, they are present only for the first 10 iterations or so usually. Maybe we can introduce verbosity levels for such messages.